src/property-filter/property-editor.tsx

```diff
@@ -43,6 +57,55 @@ export function PropertyEditorContent<TokenValue = any>({
   );
 }
 
+export function PropertyEditorContentEnum({
+  property,
+  filter,
+  value: unknownValue,
+  onChange,
+  asyncProps,
+  filteringOptions,
+  onLoadItems,
+}: {
+  property: InternalFilteringProperty;
+  filter: string;
+  value: null | string[];
+  onChange: (value: null | string[]) => void;
+  asyncProps: DropdownStatusProps;
+  filteringOptions: readonly InternalFilteringOption[];
+  onLoadItems?: NonCancelableEventHandler<LoadItemsDetail>;
+}) {
+  const valueOptions: readonly { value: string; label: string }[] = filteringOptions
+    .filter(option => option.property?.propertyKey === property.propertyKey)
+    .map(({ label, value }) => ({ label, value }));
+  const valueHandlers = useLoadItems(onLoadItems, '', property.externalProperty);
+  const value = !unknownValue ? [] : Array.isArray(unknownValue) ? unknownValue : [unknownValue];
```

Comment:
According to the function interface, `value` can only be `null` or `string[]`, so if we already checked against falsy value with `!unknownValue`, why do we need to check that it's an array?

Reply:
Oh, true. I will change it so that it is unknown in the interface